homework/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "hackyourtemperature",
+  "version": "1.0.0",
+  "description": "HYF assignment for node module",
+  "main": "server.js",
+  "scripts": {
+    "dev": "nodemon server.js",
+    "start": "node server.js"
+  },
+  "author": "Ibrahim Sahin",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.19.2",
+    "express-handlebars": "^7.1.2",
+    "node-fetch": "^3.3.2",
+    "nodemon": "^3.1.0"
+  },
```

Nodemon will only be used when developing your backend. In nodejs we can make the difference using `devDependencies`

```suggestion
  "dependencies": {
    "express": "^4.19.2",
    "express-handlebars": "^7.1.2",
    "node-fetch": "^3.3.2"
  },
  "devDependencies": {
    "nodemon": "^3.1.0"
  },
```

---

ah, I realized I was using the wrong flag. I think I am supposed to use --save-dev while installing with npm?

I installed with 'npm i --dev nodemon'.  and I didnt check the package.json. I thought it was in devDependencies.

---

Indeed, you are correct, you need to use `--save-dev` for it to go in the correct place, or the shorthand `-D`

---

Okay, thank you!